File: frontend/src/app/_models/loan.ts
export interface Loan {
  id: number;
  amount: number;
  currentBalance: number;
  isActive: number;
  createdAt: string;
  updateAt: string;
  idApplicant: number;
  idApplicantNavigation: IdApplicantNavigation;
  payments?: PaymentsEntity[] | null;
}
export interface IdApplicantNavigation {
  id: number;
  fullName: string;
  dni: string;
}
export interface PaymentsEntity {
  id: number;
  amount: number;
  idLoan: number;
  createdAt: string;
}

File: backend/src/Fundo.Applications.WebApi/Infraestructure/Models/Loan.cs
using System;
using System.Collections.Generic;

namespace Fundo.Applications.WebApi.Models
{
    public partial class Loan
    {
        public Loan()
        {
            Payments = new HashSet<Payment>();
        }

        public int Id { get; set; }
        public decimal? Amount { get; set; }
        public decimal? CurrentBalance { get; set; }
        public int? IsActive { get; set; }
        public DateTime? CreatedAt { get; set; }
        public DateTime? UpdateAt { get; set; }
        public int? IdApplicant { get; set; }

        public virtual Applicant IdApplicantNavigation { get; set; }
        public virtual ICollection<Payment> Payments { get; set; }
    }
}
